translations/inferno_devilish_plan.tex/cs.po

```diff
@@ -299,6 +335,8 @@ msgid ""
 "\\textbf{Ivor's Spell Deck:} 1 × Precision Spell card (it resolves on the first \\svg{unit_ranged} unit able to make a \\svg{unit_ranged} attack), 1 × Magic Arrow Spell card\n"
 "\n"
 msgstr ""
+"\\textbf{Ivorův Balíček Kouzel:} 1 × karta Kouzla Přesnosti (je použita na první \\svg{unit_ranged} jednotku schopnou útoku na dálku), 1 × karta Kouzla Magický Šíp\n"
+"\n"
```

Comment:
```suggestion
"\\textbf{Ivorův Balíček Kouzel:} 1 × Přesnost (je použita na první \\svg{unit_ranged} jednotku schopnou útoku na dálku), 1 × Magický Šíp\n"
"\n"
```
This might help you not-resize the art.

Reply:
I skipped this one for the sake of consistency